earth2studio/statistics/rmse.py

```diff
@@ -96,3 +96,118 @@ def __call__(
         """
         mse, output_coords = self.mean((x - y) ** 2, x_coords)
         return torch.sqrt(mse), output_coords
+
+
+class spread_skill_ratio:
```

Comment:
Can you add this as well as the previously added stats to to API page of the docs:

https://github.com/NVIDIA/earth2studio/blob/main/docs/modules/statistics.rst

Other than that, lgmt